playbooks/roles/deploy-osh/tasks/kubectl-install.yml

```diff
@@ -61,12 +63,20 @@
       stat:
         path: "{{ kubeconfig_file_path }}"
       delegate_to: localhost
+      register: new_config
+
+    - name: Find if an existing .kube/config file
+      stat:
+        path: "{{ ansible_user_dir }}/.kube/config"
+      delegate_to: localhost
+      register: old_config
 
     - name: Copy local runner kubeconfig file to deploy node
       copy:
         src: "{{ kubeconfig_file_path }}"
         dest: "{{ ansible_user_dir }}/.kube/config"
         mode: "0600"
+      when: new_config.stat.exists and (old_config.stat.exists == False or old_config.stat.mtime < new_config.stat.mtime)
```

Comment:
lint complains this line -- looks like my bad though... I think the lint wants it to "not old_config.stat.exists" instead.

Reply:
```when: x and y ``` should be rewritten to:

```
when:
  - "x"
  - "y"
```

Reply:
Will try to address in next patch.

Reply:
Don't forget the wrapping with double quotes :)

Reply:
Okay..will add double quotes in next patch